README.md

```diff
@@ -487,6 +487,16 @@ end
 
 ```
 
+### Development
+
+You can get development environment. It requires [vagrant](https://www.vagrantup.com/).
```

Comment:
I thought it can be convenient, cause not every developer has ruby environment on his own machine. e.g. for running tests in my case. Are you sure that it must be removed?